tests/Template/TemplateTest.php

```diff
@@ -102,8 +102,8 @@ public function testRender(): void
 			]
 		]);
 
-		$template = new Template('simple');
-		$this->assertSame('Test', $template->render(['slot' => 'Test']));
+		$template = new Template('template');
+		$this->assertSame('Test', $template->render(['foo' => 'Test']));
```

Comment:
`Discuss:` As we reuse `Snippet::load()` for `Template::render()`, templates now would also not allow to include `$slot` and `$slots` in the data array passed to the template. This is a breaking change, but maybe more consistent?

Reply:
I'm in general fine with this breaking change. Moving it to v6 does not sound like a bad idea to me anyway. It's quite a bit of a change in the template engine.